lib/postProcessLdesConceptualService.js

```diff
@@ -41,10 +41,19 @@ export async function processLdesDelta(delta) {
 
   for(const entry of toProcess) {
     try {
-      await updateNewLdesVersion(entry.graph.value, entry.subject.value, entry.object.value);
-      await updatedVersionInformation(entry.subject.value, entry.object.value);
-      await flagInstancesModifiedConcept(entry.graph.value, entry.subject.value, entry.object.value);
-      await ensureConceptDisplayConfigs(entry.graph.value, entry.subject.value, entry.object.value);
+      const versionedServiceGraph = entry.graph.value;
```

Comment:
Not strictly needed, but I found the repeated `entry.graph.value` a bit hard to follow.